src/kubernetes-ingress-controller/guides/using-kongingress-resource.md

```diff
@@ -14,6 +15,24 @@ proxy behavior.
 > annotation value will take precedence over a KongIngress value if both set
 > the same setting. This guide focuses on settings that can only be set using
 > KongIngress.
+{% endif_version -%}
+{% if_version gte:2.8.x -%}
+{:.note}
+> As of version 2.8, KongIngress sections other than `upstream` are
+> [deprecated](https://github.com/Kong/kubernetes-ingress-controller/issues/3018).
+> All settings in the `proxy` and `route` sections are now available with
+> dedicated annotations, and these annotations will become the only means of
+> configuring those settings in a future release. For example, if you had set
+> `proxy.connect_timeout: 30000` in a KongIngress and applied an
+> `konghq.com/override` annotation for that KongIngress to a Service, you will
+> need to instead apply a `konghq.com/connect-timeout: 30000` annotation to the
+> Service.
+> 
+> Plans are to replace the `upstream` section of KongIngress with [a new
+> resource](https://github.com/Kong/kubernetes-ingress-controller/issues/3174),
+> but this is still in development and `upstream` is not yet officially
+> deprecated.
```

Comment:
```suggestion
> deprecated yet.

```
I think the spacing might be causing Jekyll to think the `endif` is part of the note, so that might be why "Installation" is showing up in the note?